e2e/tests/addOrRemoveLegalCounsel_test.js

```diff
@@ -91,10 +91,11 @@ const assertChangeOfRepresentative = (I, index, method, respondentName, actingUs
   }
 };
 function assertLegalCounsellorWasAdded(caseViewPage, I) {
-  const LEGAL_COUNSELLOR_ORGANISATION_NAME_XPATH = `//mat-tab-body//*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="Respondents 1"]]]
-    //*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="Legal Counsellor 1"]]]
-    //*[@class="complex-panel" and .//*[@class="complex-panel-title" and .//*[text()="Organisation"]]]
-    //*[contains(@class,"complex-panel-compound-field") and .//td/span[text()="Name:"]]`;
+  const LEGAL_COUNSELLOR_ORGANISATION_NAME_XPATH = '//mat-tab-body' +
```

Comment:
There is an `I.seeOrganisationInTab` method in the `tabs.js` file, I think this would construct the same thing or something similar

Reply:
Hum. Interesting.
Will definitely check it out.
Thanks.

Reply:
Done